src/events/SDL_mouse_c.h

```diff
@@ -154,9 +154,6 @@ extern void SDL_SetMouseFocus(SDL_Window *window);
 /* Update the mouse capture window */
 extern int SDL_UpdateMouseCapture(SDL_bool force_release);
 
-/* Get the current mouse button state for a mouse */
-Uint32 SDL_GetMouseButtonState(SDL_Mouse *mouse, SDL_MouseID mouseID, SDL_bool include_touch);
```

Comment:
Is this not called outside of SDL_mouse.c?

Reply:
It's a duplication definition (it's in the public header, too).